src/praisonai-bot/praisonai_bot/gateway/server.py

```
@@ -4225,6 +4283,54 @@ async def _deliver_scheduled_result(
+                route, e,
+            )
+            return await router.deliver(route, text, idempotency_key=idem)
+
+        async def _send(target: str, payload: Dict[str, Any]) -> bool:
+            return await router.deliver(
+                target, payload.get("text", ""), idempotency_key=idem,
+            )
+
+        try:
+            succeeded, _ = await outbox.drain(_send)
+        except Exception as e:  # pragma: no cover - defensive
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)